### ISSUE TITLE
Unable to import cyrk_ode

### ISSUE THREAD

**Issue by dihm** [jrenaud90/CyRK#7]
I'm excited to try out the cython compiled ODE solver but having difficulty with what appears to be binary compatibility with numpy.

Error:
```shell
ValueError                                Traceback (most recent call last)
Input In [111], in <cell line: 2>()
      1 from numba import njit
----> 2 from CyRK import cyrk_ode

File ~\miniconda3\envs\arc\lib\site-packages\CyRK\__init__.py:9, in <module>
      6 from .nb.nbrk import nbrk_ode
      8 # Import cython solver
----> 9 from CyRK.cy.cyrk import cyrk_ode

File ~\miniconda3\envs\arc\lib\site-packages\CyRK\cy\_cyrk.pyx:1, in init CyRK.cy.cyrk()

ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
```

Relevant installed package versions:
- python=3.9.12
- numba=0.55.1
- numpy=1.21.5
- cython=0.29.32
- cyrk=0.1.1

Everything but cyrk is installed from conda standard channel.

**Comment by jrenaud90** [comment]
This appears to be a problem with that version of numpy with no obvious fix. But, I have changed the dependency requirements to:
```python
numba>=0.54.1
numpy>=1.22
scipy>=1.9.3
python>=3.8
```
and have not had any issues. These have been updated in "CyRK>=0.1.3a3" along with some other minor changes.

I also suspect there is something wrong with the PyPI wheels. So, for the time being, I would recommend reinstalling using,
`pip install -v "cyrk==0.1.3a3" --no-binary cyrk`
to force it to recompile the cython code.

Hope that helps!
